domene/src/main/java/no/nav/foreldrepenger/abonnent/pdl/kafka/AivenProperties.java

```diff
@@ -0,0 +1,148 @@
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.common.config.SaslConfigs;
+import org.apache.kafka.common.config.SslConfigs;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.errors.LogAndFailExceptionHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.confluent.kafka.serializers.KafkaAvroDeserializerConfig;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import no.nav.person.pdl.leesah.Personhendelse;
+import no.nav.foreldrepenger.konfig.KonfigVerdi;
+
+import static io.confluent.kafka.serializers.KafkaAvroDeserializerConfig.SPECIFIC_AVRO_READER_CONFIG;
+
+@Dependent
+public class AivenProperties {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PdlLeesahHendelseProperties.class);
+
+    private static final String KAFKA_AVRO_SERDE_CLASS = "kafka.avro.serde.class";
+
+    private static final Environment ENV = Environment.current();
+
+    private final String bootstrapServers;
+    private final String schemaRegistryUrl;
+    private final String applicationId;
+    private final String trustStorePath;
+    private final String keyStoreLocation;
+    private final String credStorePassword;
+    private final String clientId;
+
+    private final Topic<String, Personhendelse> topic;
+
+    private final boolean isDeployment = ENV.isProd() || ENV.isDev();
+
+    @Inject
+    public AivenProperties(@KonfigVerdi(value = "kafka.aiven.pdl.leesah.topic", defaultVerdi = "default") String topicName,
+                           @KonfigVerdi(value = "kafka.pdl.leesah.topic") String onpremTopicName,
+                           @KonfigVerdi("KAFKA_BROKERS") String bootstrapServers,
+                           @KonfigVerdi("KAFKA_SCHEMA_REGISTRY") String schemaRegistryUrl,
+                           @KonfigVerdi("KAFKA_SCHEMA_REGISTRY_USER") String schemaRegistryUsername,
+                           @KonfigVerdi("KAFKA_SCHEMA_REGISTRY_PASSWORD") String schemaRegistryPassword,
+                           @KonfigVerdi("KAFKA_TRUSTSTORE_PATH") String trustStorePath,
+                           @KonfigVerdi("KAFKA_KEYSTORE_PATH") String keyStoreLocation,
+                           @KonfigVerdi("KAFKA_CREDSTORE_PASSWORD") String credStorePassword,
+                           @KonfigVerdi(value = KAFKA_AVRO_SERDE_CLASS, defaultVerdi = "io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde") String kafkaAvroSerdeClass) {
+        this.trustStorePath = trustStorePath;
+        this.keyStoreLocation = keyStoreLocation;
+        this.credStorePassword = credStorePassword;
+        this.schemaRegistryUrl = schemaRegistryUrl;
+        this.applicationId = "fpabonnent";
+        this.clientId = "fpabonnent-" + UUID.randomUUID();
```

Comment:
Holder kanskje med fpabonnent eller UUID? - client.id er mest for logging/sporing
Dok sier at kunn application.id og bootstrap er påkrevd https://docs.confluent.io/platform/current/streams/developer-guide/config-streams.html

Reply:
Jeg forsøkte å droppe å sette den, og lokalt kunne jeg ikke se at jeg fikk generert en client.id. Satte den derfor for sikkerhets skyld. Fulgte default mønsteret, som er applicationId prefiks + uuid